configuration_example-d2-docker.json

```diff
@@ -43,7 +39,12 @@
       "trackerPrograms": [
         "q49tR53YqUX"
       ]
-    }
+    },
+      "ALL": {
```

Comment:
Where is that entry used? Theirs values `%` are a bit cryptic, is it documented?